README.md

```diff
@@ -1,7 +1,14 @@
 RISC-V Debug Specification
 ==========================
 
-A pre-built version of the spec is included [here](https://github.com/riscv/riscv-debug-spec/blob/master/riscv-debug-draft.pdf).
+This repository contains the latest draft version of the Debug Specification,
+as well as an errata document that accompanies any ratified versions of the
+Debug Specification.
+
+The following pre-built PDFs may be of interest:
+* [0.13-DRAFT, which will likely be ratified](https://github.com/riscv/riscv-debug-spec/blob/task_group_vote/riscv-debug-draft.pdf)
+* [Errata Document](https://github.com/riscv/riscv-debug-spec/blob/master/riscv-debug-errata.pdf)
+* [Latest draft](https://github.com/riscv/riscv-debug-spec/blob/master/riscv-debug-draft.pdf)
```

Comment:
Should we say something in the README about the rationale of 0.13.1?

I was assuming it was as described on: https://semver.org/
```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.
Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

```

Reply:
The spec itself should mention all previous ratified versions, and their compatibility, but not until there actually is a ratified spec.